data/preset/escola.yaml

```diff
@@ -17,22 +17,33 @@ defines:
       shortdef: 'Layout.Box(orientation: vertical)'
       slots:
         contents:
-        - shortdef: Layout.Box
+        - type: Layout.Overlay
           styles:
             - landing
           slots:
-            contents:
-            - shortdef: 'Banner.Dynamic(show-subtitle: true, mode: text, expand: true)'
-        - shortdef: 'ContentGroup.ContentGroup(expand: true)'
-          slots:
-            <<: *main-arrangement-slot
-            selection: Selection.AllSets
-        - shortdef: 'Layout.Box(orientation: vertical, halign: fill, hexpand: true)'
-          styles:
-            - footer
+            content:
+              shortdef: Layout.Box
+              styles:
+                - background
+            overlays:
+            - shortdef: 'Layout.Box(orientation: vertical, valign: center, vexpand: true)'
+              slots:
+                contents:
+                - shortdef: 'Banner.Dynamic(mode: text, expand: true)'
+                - shortdef: 'ContentGroup.StaticText(label: "TV ESCOLA")'
```

Comment:
> Isn't the app title already part of the Banner.Dynamic above this label?

Yes but this text, "TV ESCOLA" is not a title, nor a subtitle. It's just text that we want to include on all tv escola apps, and since we don't yet have a way of parameterizing yaml files, I have to just stick it in as a constant. We're not making any other non-tv-escola apps for 3.1 that use this template, so I figure by the time we need to use it for something else we'll have a way to parameterize the yaml